operator/pkg/ciliumendpointslice/namespace.go

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package ciliumendpointslice
+
+import (
+	"context"
+
+	k8s "github.com/cilium/cilium/pkg/annotation"
+	"github.com/cilium/cilium/pkg/k8s/resource"
+	slimcorev1 "github.com/cilium/cilium/pkg/k8s/slim/k8s/api/core/v1"
+	"github.com/cilium/cilium/pkg/logging/logfields"
+)
+
+const (
+	priorityNamespaceAnnotation = "cilium.io/ces-namespace"
```

Comment:
Let's document it in CES documentation.

Reply:
Docs should be updated here https://github.com/dlapcevic/cilium/blob/main/Documentation/operations/upgrade.rst#117-upgrade-notes?

Anywhere else?

Reply:
I think most important is here: https://docs.cilium.io/en/latest/network/kubernetes/ciliumendpointslice/

Reply:
I modified added a point to Configuration Options in ciliumendpointslice documentation